experimental/delegate/opt.mlir

```diff
@@ -0,0 +1,47 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(iree-preprocessing-apply-pdl-patterns{patterns-file=%p/opt.pdl.mlir}, cse)" %s | FileCheck %s
+
+// TODO: Add filecheck checks
```

Comment:
Maybe just add a simple FileCheck test... Just to show that there are two matmul calls.

Reply:
Good idea!  Will do

Reply:
Added one for single OPT matmul as well